Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -87,8 +107,126 @@ YUI.add('ez-actionbarview-tests', function (Y) {
                 location: this.view.get('location')
             });
         },
+
+        "Should instantiate sendToTrashActionView": function () {
+            this._isActionCreated('sendToTrash', {});
+        },
+
+        "Should not instantiate deleteActionView": function () {
+            this._isNotActionCreated('delete', {});
+        },
+    });
+
+    userViewTest = new Y.Test.Case({
+        name: "eZ Action Bar View For Users test ",
+
+        setUp: function () {
+            this.location = {};
+            this.content = {};
+            this.contentTypeMock = new Mock();
+
+            Y.eZ.CreateContentActionView = Y.Base.create('createContentActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {}, {
+                contentTypeGroups: {},
+                contentTypeSelectorView: {},
+                contentType: {}
+            });
+            Y.eZ.TranslateActionView = Y.Base.create('translateActionView', Y.eZ.ButtonActionView, [Y.eZ.Expandable], {}, {
+                location: {},
+                content: {}
+            });
+            Y.eZ.MoveContentActionView = Y.Base.create('moveContentActionView', Y.eZ.ButtonActionView, [], {}, {
+                location: {}
+            });
+
+            Mock.expect(this.contentTypeMock, {
+                method: 'hasFieldType',
+                args: ['ezuser'],
+                returns: true
+            });
+
+            this.view = new Y.eZ.ActionBarView({
+                location: this.location,
+                content: this.content,
+                contentType: this.contentTypeMock
+            });
+        },
+
+        tearDown: function () {
+            delete Y.eZ.CreateContentActionView;
+            delete Y.eZ.TranslateActionView;
```

Comment:
@dpobel: Ok. But this was a copy paste from the other test.should we fix it in the other too?